OsmAnd/res/values/strings.xml

```diff
@@ -36,7 +36,7 @@
     <string name="coord_input_save_as_track_descr">You added %1$s points. Type a filename and tap \"Save\".</string>
     <string name="error_notification_desc">Please send a screenshot of this notification to support@osmand.net</string>
     <string name="quick_action_edit_actions">Edit actions</string>
-    <string name="get_osmand_live">Get OsmAnd Live to unlock all features: Daily map updates with unlimited downloads, all paid and free plugins, Wikipedia, Wikivoyage and much more.</string>
+    <string name="get_osmand_live">Get OsmAnd Live to unlock all features: Daily map updates with unlimited downloads, all paid and gratis plugins, Wikipedia, Wikivoyage and much more.</string>
```

Comment:
"free" is much more commonly used than "gratis" in English.

Reply:
Free is ambiguous, gratis is not. An error in translation here is on the contrary wrong 100% of the time.

Reply:
Again, I don't think it's acceptable to use "gratis" here just to ensure that there are better translations. I strongly suspect any native English speaker would use "free" here. And while "free" has multiple meanings, the meaning here is clear from the context.

It's also important the interface uses terms which users are familiar with. In previous pull requests you changed "Christmas" to "Yuletide". Thankfully this was reverted before it was included in a release. Had it not been, I imagine OsmAnd users would be somewhat perplexed.

Using "gratis" here is not a good idea.

Reply:
The choice of precision is clearly one made by some English speakers, otherwise the use of "gratis" would end, notwithstanding, the word itself seems to be understood by at-least an acceptable demographic. The feeling of familiarity is a valid concern, but pales in comparison to the potential error. The context is a piece of software that is free software, a quality plugins could or could not have. In this case it isn't very likely, so no solution is actually ideal. Other than potentially just "all plugins", which would be my choice. Maybe making a comment if not so.

In changing "Christmas" to "Yule" (an English word derived from the indigenous season called, "yuletide") in English, consistency is found with the OsmAnd instance of "a market that takes place during this time", corrected to the effect of "yule-market", is the correct use for that particular facet of it. Thus it would be both correct and consistent. Similarly, the tale of Jesus in the stable is not one of the Yule-stories, it is a Christmas story. My rationale is representing peoples traditions accurately. "Somewhat perplexed" trumps misrepresenting culture no matter how common the error.

I make the argument, and grant changes I don't agree with.